x-pack/plugins/uptime/public/apps/plugin.ts

```diff
@@ -63,6 +67,58 @@ export class UptimePlugin
       });
     }
 
+    // This doesn't actually work
```

Comment:
Hi @andrewvc I checked out your branch and I didn't see any problem. Can you tell me why you mentioned that it doesn't work?